layouts/shortcodes/changelog.xml

```diff
@@ -0,0 +1,7 @@
+<item>
+  <title>{{ .Get "title" }}</title>
+  <link>https://docs.viam.com/appendix/#{{ .Get "title" | anchorize }}</link>
+  <pubDate>{{ .Get "date" | time.Format "Mon, 02 Jan 2006 15:04:05 -0700" }}</pubDate>
```

Comment:
Not sure how the date formatting works but, this is different from the format passed into the shortcodes in changelog.md...how does this work? And do we really need to have a date instead of just a month, when the date is not actually the first of the month?

Reply:
This is the format RSS expects. The commands change the formatting to match

Reply:
I think we do need to add a full date but I'll test tomorrow

Reply:
Okay. I don't love the idea that someone could get the RSS feed and think that they should be looking for a viam-server version that came out the first of the month

Reply:
If it's not possible to eliminate the full date day, do you think it'd be worth it to (in a separate ticket) fix these with more accurate dates? We used to have most of them in here anyway I think, so could just re-apply them